src/customStyle.ts

```diff
@@ -73,9 +67,14 @@ export function setStyles(style: Style[]) {
   }
   hasdocTypechanged = docType !== documentType;
   docType = documentType;
-  if (_view && style && hasdocTypechanged) {
-    _view.dispatch(_view.state.tr.scrollIntoView());
-    hasdocTypechanged = false;
+  if (docType) {
+    hasdocTypechanged = true;
+  }
+  if (style[0] === undefined ||!Object.prototype.hasOwnProperty.call(style[0], 'docType')) {
```

Comment:
Use [Object.hasOwn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn) instead of Object.protoype.hasOwnProperty.